webapp.rb

```
@@ -33,6 +37,13 @@ class WebApp < Sinatra::Base
       secure: production?,        # 本番環境ではHTTPS接続のみでCookieを送信
       same_site: :lax,           # クロスサイトリクエスト時の挙動を制御（CSRF緩和）
       secret: settings.session_secret
+
+    set :projects, [
+      { id: 1, name: "Customer Discovery", owner: "Product", status: "In Progress", description: "Interview and synthesize pain points before the next planning cycle." },
+      { id: 2, name: "Design System Refresh", owner: "Design", status: "Planned", description: "Lightweight refresh of spacing, color tokens, and form components." },
+      { id: 3, name: "Metrics Dashboard", owner: "Data", status: "Blocked", description: "Ship a self-serve KPI dashboard; blocked waiting on tracking plan." }
```

RuboCop (1.81.7)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)